pkg/ruler/ruler.go

```diff
@@ -59,6 +60,8 @@ var (
 type Config struct {
 	// This is used for template expansion in alerts; must be a valid URL.
 	ExternalURL flagext.URLValue `yaml:"external_url"`
+	// Config parameters for the GRPC Client
+	GRPCClientConfig grpcclient.Config `yaml:"grpc_client_config"`
```

Comment:
From the UX perspective, we're adding all `grpcclient.Config` to the ruler client just for TLS.